### DISCUSSION TITLE
Calculator

### DISCUSSION THREAD

**Discussion by brouty** [NickvisionApps/Denaro#712] [Ideas]
Would be nice to open a simple calculator from within the app.
Thanks for a great app. It is exactly what I needed.

**Comment by nlogozzo** [comment]
DenaroOS 😅

I mean I don't really know how'd we integrate it into the app.

Cause it's either you open Denaro' calculator which would be a separate dialog or just open your system's calculator app, which would just be another dialog.

So I don't see the benefit of reinventing the wheel for that.

**Reply by fsobolev** [reply]
One can also enable Calculator search provider in GNOME Settings and calculate things by typing after opening activites overview. Denaro is not the only app where users need to calculate things sometimes, and the existence of fast way to do this using the desktop environment's functionality eliminates the need to reimplement calculator in every app. It's sad that not every DE has this.

> DenaroOS 😅

Denaro becoming new Emacs 😂
